config/parser.go

```diff
@@ -58,6 +58,13 @@ func ParserFromViper(v *viper.Viper) *Parser {
 	return &Parser{v: v}
 }
 
+func NewParserFromStringMap(data map[string]interface{}) *Parser {
+	v := NewViper()
+	// Cannot return error because the subv is empty.
+	_ = v.MergeConfigMap(cast.ToStringMap(data))
```

Comment:
Can't we avoid the cast here?
```suggestion
	_ = v.MergeConfigMap(data)
```

I think the cast function does nothing here https://github.com/spf13/cast/blob/1ffadf551085444af981432dd0f6d1160c11ec64/caste.go#L983-L984

Reply:
Good call.